src/main/Catalog.js

```diff
@@ -1,27 +1,29 @@
 import React, { useState } from "react";
 
+const url = "https://api.punkapi.com/v2/beers";
+
 function Catalog() {
   let data;
```

Comment:
I don't think you need `data` in this component. You can simply use local `response` constant and pass it to the `setBeerList`.